gobblin-core/src/main/java/gobblin/instrumented/writer/InstrumentedDataWriterDecorator.java

```diff
@@ -89,6 +89,15 @@ public long bytesWritten()
   }
 
   @Override
+  public State getFinalState() {
```

Comment:
Seems you can just do `this.getFinalState()` like you do for other `Decorator` classes.